### PR TITLE
DOC/ENH: add a "rules" section to the docs

### PR DESCRIPTION
**Pull request by fmaussion**
Fixes https://github.com/ASPP/pelita/issues/476 

Related: did you guys ever tried to change the winning condition to eating all the enemy's food first regardless of how many times you die? I guess this would change the strategies quite a bit, but for good or for bad?

### CODE REVIEW

**Comment by Debilski** [comment]
We probably thought about it (I think sometimes when giving the presentation, I still forget to explain that eating all food doesn’t automatically win the game for you). It might make some of the sitting-on-food strategies invalid, so this could be an advantage.
On the other hand, it is unclear why we should be having points then in the first place.

**Comment by fmaussion** [comment]
> forget to explain that eating all food doesn’t automatically win the game for you

This is the purpose of this PR ;)

**Comment by otizonaizit** [comment]
I think winning by eating all the food would restrict the number of strategies, in the sense that then it becomes "just" a matter of finding the sequence of food pellets which can be reached through the minimal number of steps, i.e. it turns it into a mathematical problem (which may already have a solution?). Giving points for the killing (and a lot at that) makes the solution space way more complicated, and makes it also more fun to watch, because students will then try to kill ;)

File: doc/source/rules.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Eating**: when a Bot eats a food pellet, the food is permanently removed and
+*one point* is scored for that Bot’s team.
+
+**Eating another Bot**: when a Bot is eaten by an opposing destroyer, it
+returns to its starting position (as a harvester). *5 points*
+are awarded for eating an opponent.
+
+**Observations**: Bots can only observe an opponent’s exact position, if they
+or their teammate are within 5 squares of the opponent bot. If they are
+further away, the opponent’s positions are noised.
```
Comment (otizonaizit) [inline doc/source/rules.rst]: We should say here that 5 squares does *not* mean that they are reachable in 5 moves, i.e. it is manhattan distance and not distance on the graph (students are often confused by this). And we should specify what kind of noise is added (I think it is a uniformly distributed integer between -5 and +5 on each coordinate? @Debilski ?)

File: doc/source/rules.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+**Timeout**: each Player only has 3 seconds to return a valid move. If it
+doesn’t, a random move is executed and all later return values are
+discarded. 5 timeouts and you’re out!
+
+**Winning**: the game ends when *either* one team eats all of the opponents’
+food pellets, *or* after 300 rounds. The team with more points at the end of
+the game wins.
```
Comment (otizonaizit) [inline doc/source/rules.rst]: This section could be slipt in: **End**, the game ends... and **Winning**. This way it is also visually clear that winning the game or causing a game over is not the same....

Comment (fmaussion) [reply doc/source/rules.rst]: Good idea. Done.

**Pull request merged**